app/search_builders/sufia_search_builder.rb

```diff
@@ -0,0 +1,9 @@
+class SufiaSearchBuilder < Blacklight::Solr::SearchBuilder
```

Comment:
Do we need a new file here, or can this code be added to `app/search_builders/sufia/search_builder.rb`?